DFEAGILEDEVOPS/MTC
# 12349 Post request for question data

Specification and documentation around this can be found here:
https://app.swaggerhub.com/apis/GuyHarwood/mtc-admin/1.1.0

Steps for testing:
- Run npm start on admin directory
- Use postman , paw or any other app that allows HTTP post request examination
- Add localhost:3001/api/questions as URL
- Request header Content Type should be application-json
- Payload should be a json object with pupil id and schoolId i.e. 
`{
  "schoolPin": "fdh6fb3h",
  "pupilPin": "d55sg"
}
`
